cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -54,11 +57,27 @@ type PolicyProcessor struct {
 	Out                       io.Writer
 }
 
+type PolicyExceptionLister struct {
+	Exceptions []kyvernov2beta1.PolicyException
+}
+
+func (l *PolicyExceptionLister) List(selector labels.Selector) ([]*kyvernov2beta1.PolicyException, error) {
+
+	var matchedExceptions []*kyvernov2beta1.PolicyException
+	for i := range l.Exceptions {
+		matchedExceptions = append(matchedExceptions, &l.Exceptions[i])
+	}
+	return matchedExceptions, nil
```

Comment:
So `List()` should get exceptions based on selectors but instead you are returning the exceptions loaded from files or whatever the source. Right?

Reply:
yes i should get exception based on selector okay i should correct it like 
```
func (l *PolicyExceptionLister) List(selector labels.Selector) ([]*kyvernov2beta1.PolicyException, error) {

	var matchedExceptions []*kyvernov2beta1.PolicyException
	for i := range l.Exceptions {
		exceptionLabels := labels.Set(l.Exceptions[i].GetLabels())
		if selector.Matches(exceptionLabels) {
			matchedExceptions = append(matchedExceptions, &l.Exceptions[i])
		}
	}
	return matchedExceptions, nil
}

```